packages/node/src/sdk/index.ts

```diff
@@ -140,7 +140,7 @@ function _init(
   const scope = getCurrentScope();
   scope.update(options.initialScope);
 
-  if (options.spotlight && options.integrations.some(({ name }) => name !== SPOTLIGHT_INTEGRATION_NAME)) {
+  if (options.spotlight && !options.integrations.some(({ name }) => name === SPOTLIGHT_INTEGRATION_NAME)) {
```

Comment:
We can also do
```suggestion
  if (options.spotlight && options.integrations.all(({ name }) => name !== SPOTLIGHT_INTEGRATION_NAME)) {
```

But I think the current version expresses the intent a bit more clearly: if there are no integrations matching this name, add it.

Reply:
I'm fine with any of these, as long as we only add the integration if it isn't yet present 👍 
Thanks for fixing!